### DISCUSSION TITLE
Home of org.eclipse.persistence.internal.weaving.PersistenceWeaved

### DISCUSSION THREAD

**Discussion by mebigfatguy** [eclipse-ee4j/eclipselink#2241] [General]
Hi folks, 

Sorry i have been flumoxed so far, but i'm looking for where the class

org.eclipse.persistence.internal.weaving.PersistenceWeaved

is found in the "split up eclipselink jars". I see it in the fat eclipselink.jar, but no where else. Can someone point me to what jar that would be?

**Comment by rfelcman** [comment]
Yes in _eclipselink.jar_ bundle and `org.eclipse.persistence:org.eclipse.persistence.core` (Maven coordinate).

**Comment by mebigfatguy** [comment]
thank you. .stupid of me.
